### PR TITLE
Fix get_items

### PR DESCRIPTION
**Pull request by dyashuni**
Add `return_type `to `get_items`
Fixes https://github.com/nmslib/hnswlib/issues/223

### CODE REVIEW

File: tests/python/bindings_test_replace.py [hunk 1]
```diff
@@ -94,10 +94,10 @@ def testRandomSelf(self):
         remaining_data = comb_data[remaining_labels_list]
 
         returned_items = hnsw_index.get_items(remaining_labels_list)
-        self.assertSequenceEqual(remaining_data.tolist(), returned_items)
+        self.assertTrue((remaining_data == returned_items).all())
```
Comment (yurymalkov) [inline tests/python/bindings_test_replace.py:97]: Maybe add tests to check the output type?

Comment (dyashuni) [reply tests/python/bindings_test_replace.py:97]: I added type check for return results of get_items to bindings_test_getdata.py

**Comment by yurymalkov** [comment]
Thank you!

**Comment by eddie-scio** [comment]
Hey @yurymalkov -- would love to get this fix into a new pypi release as the associated latency wins unblock some modeling changes on our end.  Is that something lightweight to do, or can we help in any way?  Thanks!

**Comment by yurymalkov** [comment]
@eddie-scio will try to make a release soon (after merging/reviewing last 4 PRs)

**Comment by eddie-scio** [comment]
Thank you, Yury!

**Pull request merged**